Project/UISS/UISSPropertySetter.m

```diff
@@ -71,13 +71,16 @@ - (SEL)findSelectorMatchingRegexp:(NSString *)regexp class:(Class <UIAppearance>
     unsigned int count = 0;
     Method *methods = class_copyMethodList(class, &count);
 
-    NSString *selectorPrefix = self.selectorPrefix;
+    const char *selectorPrefix = self.selectorPrefix.UTF8String;
 
     for (int i = 0; i < count; i++) {
         SEL selector = method_getName(methods[i]);
-        NSString *selectorString = NSStringFromSelector(selector);
 
-        if ([selectorString hasPrefix:selectorPrefix]) { // reducing the use of regular expression for performance reasons
+        BOOL hasPrefix = strncmp(selectorPrefix, sel_getName(selector), strlen(selectorPrefix)) == 0;
+
+        if (hasPrefix) { // reducing the use of NSString & regular expression for performance reasons
+            NSString *selectorString = NSStringFromSelector(selector);
```

Comment:
How converting results of `sel_getName` you are using above to `NSString *` instead of `NSStringFromSelector`? Not sure, but it might be faster.